invenio_communities/assets/semantic-ui/js/invenio_communities/community/communities-carousel/CarouselItem.js

```diff
@@ -12,45 +12,53 @@ import _truncate from "lodash/truncate";
 import { i18next } from "@translations/invenio_communities/i18next";
 import { Image } from "react-invenio-forms";
 import { Grid, Header, Item, Button } from "semantic-ui-react";
+import Overridable from "react-overridable";
 
-export default class CarouselItem extends Component {
+class CarouselItem extends Component {
   render() {
     const { community, defaultLogo, className } = this.props;
 
     return (
-      <Item
-        className={`carousel flex align-items-center ${className}`}
-        key={community.id}
+      <Overridable
+        id="CarouselItem.layout"
```

Comment:
I don't know if it was discussed or documented, but I think this ID is not unique enough... IDs for reusable components are not a great idea, because if I use the same cmp 2 times, I cannot customize the 2 cmps differently...
Ping @kpsherva we should maybe document best practices around overridable IDs...

Reply:
indeed, it should be prefixed with the module name, i.e. `InvenioCommunities.CarouselItem.layout`